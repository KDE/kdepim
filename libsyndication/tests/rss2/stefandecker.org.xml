<?xml version="1.0" encoding="utf-8" ?>
<!-- Source: www.stefandecker.org
     Properties:
      - excessive comments information (<comments>, <wfw:comment>, <wfw:commentRss>, <slash:comments>)
      - use of creative Commons vocab for licensing information
      - "email (name)" author info
    -->
<rss version="2.0"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:admin="http://webns.net/mvcb/"
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
   xmlns:wfw="http://wellformedweb.org/CommentAPI/"
   xmlns:content="http://purl.org/rss/1.0/modules/content/"
   xmlns:creativeCommons="http://backend.userland.com/creativeCommonsRssModule">
<channel>
    <title>Stefan Decker on the Semantic Web</title>
    <link>http://www.stefandecker.org/blog/</link>
    <description>A weblog covering semantics in computer science</description>
    <dc:language>en</dc:language>
    
    <generator>Serendipity 0.9 - http://www.s9y.org/</generator>
    <pubDate>Fri, 16 Dec 2005 11:00:27 GMT</pubDate>

    <image>
        <url>http://www.stefandecker.org/blog/templates/default/img/s9y_banner_small.png</url>
        <title>RSS: Stefan Decker on the Semantic Web - A weblog covering semantics in computer science</title>
        <link>http://www.stefandecker.org/blog/</link>
        <width>100</width>
        <height>21</height>
    </image>
<item>
    <title>Semantic Web 2.0 Tutorial at World Wide Web Conference in Edinburgh</title>
    <link>http://www.stefandecker.org/blog/archives/13-Semantic-Web-2.0-Tutorial-at-World-Wide-Web-Conference-in-Edinburgh.html</link>
    <comments>http://www.stefandecker.org/blog/archives/13-Semantic-Web-2.0-Tutorial-at-World-Wide-Web-Conference-in-Edinburgh.html#comments</comments>
    <wfw:comment>http://www.stefandecker.org/blog/wfwcomment.php?cid=13</wfw:comment>
    <slash:comments>0</slash:comments>
    <wfw:commentRss>http://www.stefandecker.org/blog/rss.php?version=2.0&amp;type=comments&amp;cid=13</wfw:commentRss>
    <author>nospam@example.com (Stefan Decker)</author>
    <content:encoded>
&lt;p&gt;I just got the notification that our tutorial proposal (together with &lt;a href=&quot;http://www.johnbreslin.com/&quot;&gt;John Breslin&lt;/a&gt;) for the &lt;a href=&quot;http://www2006.org/&quot;&gt;World Wide Web Conference 2006&lt;/a&gt; on &amp;quot;Semantic Web 2.0: Creating Social Semantic Information Spaces&amp;quot; has been accepted. The contents of the tutorial will be about Semantic Blogging, Semantic Wikis, Social Semantic Collaborative Filtering and other new approaches related to applying and deploying semantics andÂ social networks for information dissemination and assessment. We will introduce the audience in the state of the art in this area, including the approaches that are developed in the Semantic Web cluster at the Digital Enterprise Research Institute. The tutorial is scheduled for Friday, May 26, 2006 in Edinburgh, at the WWW Conference.&lt;/p&gt;    </content:encoded>
                
    <pubDate>Sat, 10 Dec 2005 14:19:01 +0100</pubDate>
    <guid isPermaLink="false">http://www.stefandecker.org/blog/archives/13-guid.html</guid>
    <creativeCommons:license>http://creativecommons.org/licenses/by-nc-sa/2.5/</creativeCommons:license></item>

</channel>
</rss>
