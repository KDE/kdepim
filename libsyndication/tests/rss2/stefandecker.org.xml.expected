# Feed begin ######################
title: #Stefan Decker on the Semantic Web#
link: #http://www.stefandecker.org/blog/#
description: #A weblog covering semantics in computer science#
language: #en#
# Image begin #####################
url: #http://www.stefandecker.org/blog/templates/default/img/s9y_banner_small.png#
title: #RSS: Stefan Decker on the Semantic Web - A weblog covering semantics in computer science#
link: #http://www.stefandecker.org/blog/#
height: #21#
width: #100#
# Image end #######################
# Item begin ######################
id: #http://www.stefandecker.org/blog/archives/13-guid.html#
title: #Semantic Web 2.0 Tutorial at World Wide Web Conference in Edinburgh#
link: #http://www.stefandecker.org/blog/archives/13-Semantic-Web-2.0-Tutorial-at-World-Wide-Web-Conference-in-Edinburgh.html#
content: #<p>I just got the notification that our tutorial proposal (together with <a href="http://www.johnbreslin.com/">John Breslin</a>) for the <a href="http://www2006.org/">World Wide Web Conference 2006</a> on &quot;Semantic Web 2.0: Creating Social Semantic Information Spaces&quot; has been accepted. The contents of the tutorial will be about Semantic Blogging, Semantic Wikis, Social Semantic Collaborative Filtering and other new approaches related to applying and deploying semantics and social networks for information dissemination and assessment. We will introduce the audience in the state of the art in this area, including the approaches that are developed in the Semantic Web cluster at the Digital Enterprise Research Institute. The tutorial is scheduled for Friday, May 26, 2006 in Edinburgh, at the WWW Conference.</p>#
datePublished: #Sat Dec 10 14:19:01 2005#
dateUpdated: #Sat Dec 10 14:19:01 2005#
# Person begin ####################
name: #Stefan Decker#
email: #nospam@example.com#
# Person end ######################
commentsCount: #0#
commentsLink: #http://www.stefandecker.org/blog/archives/13-Semantic-Web-2.0-Tutorial-at-World-Wide-Web-Conference-in-Edinburgh.html#comments#
commentsFeed: #http://www.stefandecker.org/blog/rss.php?version=2.0&type=comments&cid=13#
commentPostUri: #http://www.stefandecker.org/blog/wfwcomment.php?cid=13#
# Item end ########################
# Feed end ########################

